src/tasks/YingTuoTask.py

```
@@ -67,7 +69,7 @@ def find_normal_challenge(self):
             return None
         self.target = get_world_map_text(self.lang, self.yingtuo_list[self.index])
         self.log_info(f"寻找{self.target}关卡")
-        for _ in range(3):  # 尝试多次寻找，增加成功率
+        for _ in range(len(self.yingtuo_list)//2):  # 尝试多次寻找，增加成功率
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)